nix/package.nix

```
@@ -96,28 +56,20 @@ assert withGLES -> stdenv.hostPlatform.isLinux; let
   in
     !(
       inRootDir
-      && (baseName == "docs" || baseName == ".github" || baseName == ".git" || baseName == "target")
+      && (baseName == "docs" || baseName == ".github" || baseName == "target")
```

```suggestion
      && (lib.any (x: baseName == x) [ "docs" "target" "nix" "flake.nix" "flake.lock" ])
```
We're (hopefully) never gonna use .github again. On the other hand, let's not drag all the Nix code into the build sandbox. This filter should probably be a fileset instead, but eh, future work.